_docs/style/spacing.md

```diff
@@ -173,150 +182,180 @@ Priyanka’s framework provides a solution for this using header and leaf-node m
-<embed src="{{site.url}}/assets/images/style/spacing/spacing-system-lists.png" width="531"/>
+
+{% figure [caption: "Drop-down selector spacing"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/style/spacing/spacing-content-lists-dropdown.png" width="239"/>
 {% endfigure %}
+
 {% endcolumn %}
 
 </div>
 
+</section>
+
+<section class="static-section" markdown="1">
+
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-#### Full-width items in a bound container
-
-A modification is made for list items that extend to the left and right edge of a visually bound container. The example is items in a drop-down menu.
+#### Paragraphs
 
-In cases like this the container does not receive a square inset. To move the first item away from the top edge of the container, the first item gets a top space of four pixels.
+Use a 24px line height in paragraphs that contain 16px font text. Add 16px spacing below a paragraph. These values were determined through visual explorations and input from the WCAG SC 1.4.8 guidelines.
 
 {% endcolumn %}
 
 {% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-{% figure [caption: "Full-width items in a bound container"][class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/style/spacing/spacing-system-full-width-items-in-a-bound-container.png" width="536"/>
+
+{% figure [caption: "Paragraph spacing"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/style/spacing/spacing-content-paragraphs.png" width="550"/>
 {% endfigure %}
+
 {% endcolumn %}
 
 </div>
 
-<div class="hxRow"  markdown="1">
+</section>
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+<section class="static-section" markdown="1">
 
-#### Form elements
+<div class="hxRow"  markdown="1">
 
-We found that eight pixels of vertical space between consecutive, stacked inputs works best.  Eight pixels of space also applies to consecutive, horizontally inline inputs.
+{% column left:"hxCol hxSpan-12" %}
 
-{% endcolumn %}
+## Component spacing
+
+Components such as buttons, text inputs, and date pickers are often placed beside each other on a control panel page. The Helix spacing system uses multiples of 4px as the baseline spacing value.
 
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-{% figure [caption: "Form element leaf-node spacing"][class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/style/spacing/spacing-system-forms.png" width="462"/>
-{% endfigure %}
 {% endcolumn %}
 
 </div>
 
-<div class="hxRow"  markdown="1">
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-#### Paragraphs
+**Spacing inside components**: Within a component, use 8px spacing for the left and right padding.
```

Comment:
is this correct?  it doesn't match the visual... but i cant remember...
let/s just double check the values.

also, the visuals here really need to match what you mention in the examples. so we shoudl include two tightly coupled inputs with 4 px between them in the visual

Reply:
Comment 1: asked in content channel whether it's 8px or 12px.
Comment 2: asked roman to address this.